website/docs/sdks/index.md

```diff
@@ -115,6 +113,8 @@ Here's some of the fantastic work our community has done to make Unleash work in
 - [silvercar/unleash-client-kotlin](https://github.com/silvercar/unleash-client-kotlin) (Kotlin)
 - [Stogon/unleash-bundle](https://git.stogon.io/Stogon/unleash-bundle/) (PHP - Symfony)
 - [uekoetter.dev/unleash-client-dart](https://pub.dev/packages/unleash) (Dart)
+- [nunogois/proxy-client-solid](https://github.com/nunogois/proxy-client-solid) (Solid)
+- [nunogois/proxy-client-react-native](https://github.com/nunogois/proxy-client-react-native) (React Native / Expo)
```

Comment:
I think you should move these two lines a bit further up, so that the list stays sorted alphabetically. You should also place the react-native client before the solid client for the same reason ☺️